kv/utils_test.go

```diff
@@ -18,39 +18,76 @@ import (
 	"strconv"
 	"testing"
 
-	. "github.com/pingcap/check"
+	"github.com/stretchr/testify/assert"
 )
 
-func TestT(t *testing.T) {
-	CustomVerboseFlag = true
-	TestingT(t)
```

Comment:
Is all the tests translated in this PR?
After removing `TestingT`,  the testsuites will not run...

Reply:
Yes. The suite itself is removed as migrated.